### PR TITLE
add ITreeMan's hackbar

### PR DESCRIPTION
**Pull request by ITreeMan**
First time hacktoberfest

### CODE REVIEW

**Review by tasnimzotder — CHANGES_REQUESTED** [review]
hey @ITreeMan, thanks for your interest. With your PR some unwanted files were added.
Delete the following files and make a commit.

1. `.DS_Store`
1. `_hackbars/.DS_Store`
1. `assets/.DS_Store`
1. `assets/images/.DS_Store`

**Comment by ITreeMan** [comment]
Sorry.  I delete the following files and make a new commit.

**Comment by tasnimzotder** [comment]
You did that right.
You just need to delete another file `_hackbars/.DS_Store`

**Comment by ITreeMan** [comment]
I'm delete _hackbars/.DS_Store

File: _hackbars/ITreeMan.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+layout: hackbar
+title: First Hacktober
+author: ITreeMan
+---
+
+Open source is a best way for developers and everybody. Thank you Open soure Dev.
+
+---
+
+## <!-- Your MESSAGE on climate change -->
```
Comment (tasnimzotder) [inline _hackbars/ITreeMan.md]: I did not notice it.

Can you add your __MESSAGE__ on climate change?

**Comment by ITreeMan** [comment]
I add my MESSAGE on climate change.

**Review by tasnimzotder — APPROVED** [review]

**Comment by tasnimzotder** [comment]
Thanks for your contributions :)

**Pull request merged**